frontend/src/components/widgets/Multiselect/Multiselect.tsx

```diff
@@ -62,10 +62,18 @@ interface MultiselectOption {
 class Multiselect extends React.PureComponent<Props, State> {
   private readonly formClearHelper = new FormClearHelper()
 
+  maxSelections: number = this.props.element.maxSelections
+
   public state: State = {
     value: this.initialValue,
   }
 
+  public overMaxSelections(): boolean | undefined {
```

Comment:
Is there any particular reason for sometimes returning `undefined` rather than `false`?

Reply:
This is, I would mainly say, undefined for when max_selections is not passed in. Aka a regular `st.multiselect`.

Reply:
But why is that actually necessary? If `max_selections` isn't passed to multiselect, this function should always return false, because we can't be over the max if there is no max. afaict the code functionally treats the current undefined case as if it was false.

Reply:
Ah I see! I should've seen that refactor. Thanks. Fixed!